src/nyc_trees/apps/core/mail/backends/boto.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import boto
+import json
+import logging
+import traceback
+
+from django.conf import settings
+from django.core.mail.backends.base import BaseEmailBackend
+
+from django_statsd.clients import statsd
+
+logger = logging.getLogger(__name__)
+
+
+class EmailBackend(BaseEmailBackend):
+
+    def __init__(self, fail_silently=False, aws_access_key_id=None,
+                 aws_secret_access_key=None, aws_region=None):
+        super(EmailBackend, self).__init__(fail_silently=fail_silently)
+        self.aws_access_key_id =\
+            aws_access_key_id or settings.EMAIL_AWS_ACCESS_KEY_ID
+        self.aws_secret_access_key =\
+            aws_secret_access_key or settings.EMAIL_AWS_SECRET_ACCESS_KEY
+        self.aws_region =\
+            aws_region or settings.EMAIL_AWS_REGION
```

Comment:
Why allow passing these values in via the `__init__` method, instead of only using them from `settings`?

We won't be be explicitly instantiating this backend, right?